radio/src/gui/common/stdlcd/radio_hardware.cpp

```diff
@@ -528,11 +528,21 @@ void menuRadioHardware(event_t event)
 
 #if !defined(PCBX9D) && !defined(PCBX9DP) && !defined(PCBX9E)
       case ITEM_RADIO_HARDWARE_INTERNAL_MODULE:
-        g_eeGeneral.internalModule =
-            editChoice(HW_SETTINGS_COLUMN2, y, STR_INTERNAL_MODULE,
-                       STR_INTERNAL_MODULE_PROTOCOLS,
-                       g_eeGeneral.internalModule, MODULE_TYPE_NONE,
-                       MODULE_TYPE_MAX, attr, event, isInternalModuleSupported);
+        reusableBuffer.radioHardware.internalModule =
```

Comment:
If you put brackets around the case like this:
```
case ITEM_RADIO_HARDWARE_INTERNAL_MODULE: {
   .... [you code] ...
} break;
```

You should not need the reusable buffer, which makes only sense if the data is shared between multiple places, which is clearly not the case.

Reply:
Thanks, that's gone now. Really shouldn't have been writing code at 1AM :-O